.travis.yml

```diff
@@ -30,7 +37,8 @@ matrix:
 
 before_script:
   - composer self-update
-  - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi;'
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
+  - if [ "$deps" = "low" ]; then composer update --prefer-lowest; fi
   - composer update --prefer-source
```

Comment:
this should not be done on the deps=low, otherwise the deps won't be low anymore